aiida_quantumespresso/parsers/parse_raw/cp.py

```diff
@@ -204,112 +235,176 @@ def parse_cp_xml_output(data):
     parsed_data = {}
 
     #CARD HEADER
-    parsed_data = copy.deepcopy(xml_card_header(parsed_data, dom))
+    try:
```

Comment:
I suggest to remove all the try/except blocks, move the parsing of the old XML (legacy) in a file raw_cp_legacy, and call it only depending on the XML version.

Reply:
I found that they were just harmless wrecks of some ancient attempt to write a happy parser. Removed